includes/admin/load.php

```diff
@@ -393,3 +396,37 @@ static function ( $name ) {
 		);
 	}
 }
+
+/**
+ * Callback function that print plugin progress indicator script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_progress_indicator_script() {
+	$js_function = <<<JS
+		function addPluginProgressIndicator( message ) {
+			document.addEventListener( 'DOMContentLoaded', function () {
+				document.addEventListener( 'click', function ( event ) {
+					if (
+						event.target.classList.contains(
+							'perflab-install-active-plugin'
+						)
+					) {
+						const target = event.target;
+						target.classList.add( 'updating-message' );
+						target.textContent = message;
```

Comment:
Ideally we would also use wp.a11y.speak

Reply:
Added in https://github.com/WordPress/performance/pull/1190/commits/4229c4fa4ee1da0763879dd8f10685ada7c27276